modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/fixstainmount/v0.1.0.json

```diff
@@ -0,0 +1,73 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpactsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/fixstainmount/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0",
+    "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [ {
+    "@id" : "this:FixStainMountShape",
+    "@type" : "sh:NodeShape",
+    "label" : "Fixing, Staining and Mounting activities description",
+    "targetClass" : "nsg:FixStainMount",
```

Comment:
I would suggest to align the type here with the type (FixationStainingMounting) reference in the [prov template](https://github.com/BlueBrain/nexus-bbp-domains/blob/master/docs/src/main/paradox/assets/provtemplates/morphology-reconstruction-prov-template.svg).